.ci/fuzz.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+# compile
+command -v clang &> /dev/null
+
+make clean
+clang \
```

Comment:
It would be better to integrate fuzzing specific rules into the existing `Makefile`, so that we don't have to maintain.

Reply:
I tried in my initial prototyping, and it failed to compile properly and broke the existing makefile functionalities. 

I will revisit this point later.

Reply:
Check 'sanitizer' target in jitboy: https://github.com/sysprog21/jitboy/blob/master/Makefile#L49-L51

Reply:
Will do it later! Thanks for the pointer.

Reply:
Rather than creating a separate standalone fuzzer program, we can explore the option of integrating the fuzzing components as a feature activated by a command-line option. This approach aims to minimize the necessary changes for fuzzer integration. If we opt for this method, we will need to address the linkage between the C++-based LibFuzzer and the existing C codebase.

Reply:
See `src/feature.h` for the use of macro `RV32_HAS`.